cogs/whisper.py

```diff
@@ -0,0 +1,26 @@
+import aiohttp
+import discord
+from discord.ext import commands
+
+class Whisper(commands.Cog):
+    """Whispers, for DCU Bot."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.session = aiohttp.ClientSession(loop=self.bot.loop)
+        # self.client = commands.Bot(command_prefix="whisper")
```

Comment:
no need to keep this line

Reply:
Thanks for pointing that out - I'll delete it for the next commit.